readthedocs/core/static-src/core/js/doc-embed/utils.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Create and return DOM nodes with given attributes.
+ *
+ * @param {String} nodeName name of the node
+ * @param {Object} attributes obj of attributes to be assigned to the node
+ * @return {Object} DOM node
```

Comment:
Just a note for the future: your syntax is valid jsdoc tag syntax, but adding a hyphen makes it a bit easier for humans to scan/read:

```suggestion
 * @param {String} nodeName - name of the node
 * @param {Object} attributes - obj of attributes to be assigned to the node
 * @return {Object} DOM node
```

Reply:
Doesn't work with `:` as well? It looks nicer to me and facilitate the reading as well.

Reply:
I'm also +1 on `:` instead of `-`. Checking if there is any style for that...

Reply:
> If you provide a description, you can make the JSDoc comment more readable by inserting a hyphen before the description. Be sure to include a space before and after the hyphen.

From https://jsdoc.app/tags-param.html

Reply:
@stsewd seems this merged without this, just so you know.

Reply:
Got it fixed in https://github.com/readthedocs/readthedocs.org/pull/9547